source/common/request_info/request_info_impl.h

```diff
@@ -15,39 +17,80 @@ struct RequestInfoImpl : public RequestInfo {
 
   RequestInfoImpl(Http::Protocol protocol) : RequestInfoImpl() { protocol_ = protocol; }
 
-  // RequestInfo::RequestInfo
-  SystemTime startTime() const override { return start_time_; }
+  const SystemTime& startTime() const override { return start_time_; }
+
+  const MonotonicTime& startTimeMonotonic() const override { return start_time_monotonic_; }
+
+  const Optional<MonotonicTime>& lastDownstreamRxByteReceived() const override {
+    return last_rx_byte_received_;
```

Comment:
Please rename all the members to match their corresponding function names, for clarity.  Eg last_downstream_rx_byte_received_ here.